index.html

```diff
@@ -14,41 +14,251 @@
-                    contact me.</p>
-                <p class="social-media-title">LET'S CONNECT</p>
-
-                <ul class="social-media-icons">
-                    <li>
-                        <a href="#" target="_blank"><i class="fa-solid fa-hand-peace"></i></a>
-                    </li>
-                    <li><a href="https://twitter.com/lmunozm" target="_blank"><i class="fa-brands fa-twitter"></i></a>
-                    </li>
-                    <li><a href="https://www.linkedin.com/in/luis-munoz-manriquez/" target="_blank"><i
-                                class="fa-brands fa-linkedin-in"></i></a>
-                    </li>
-                    <li><a href="https://github.com/lmunozm1702" target="_blank"><i class="fa-brands fa-github"></i></a>
-                    </li>
-                    <li><a href="https://medium.com/@lmunozm1702" target="_blank"><i
-                                class="fa-brands fa-medium"></i></a>
-                    </li>
+
+    <header class="toolbar-dark-compact">
+        <div class="logo-placeholder"><a href="/"><img src="assets/images/my_logo.png" alt="Portal logo"></a></div>
+        <div class="normal-button">
+            <nav><i class="fa-solid fa-bars"></i></nav>
+        </div>
+    </header>
+    <main>
+        <div class="container"></div>
+        <section class="main-content">
+            <h1>I’m Luis Glad to see you!</h1>
+            <p class="main-normal-text">I’m a software developer! I can help you build a product , feature or
+                website Look through some of
+                my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <p class="social-media-title">LET'S CONNECT</p>
+
+            <ul class="social-media-icons">
+                <li>
+                    <a href="#" target="_blank"><i class="fa-solid fa-hand-peace"></i></a>
+                </li>
+                <li><a href="https://twitter.com/lmunozm" target="_blank"><i class="fa-brands fa-twitter"></i></a>
+                </li>
+                <li><a href="https://www.linkedin.com/in/luis-munoz-manriquez/" target="_blank"><i
+                            class="fa-brands fa-linkedin-in"></i></a>
+                </li>
+                <li><a href="https://github.com/lmunozm1702" target="_blank"><i class="fa-brands fa-github"></i></a>
+                </li>
+                <li><a href="https://medium.com/@lmunozm1702" target="_blank"><i class="fa-brands fa-medium"></i></a>
+                </li>
+            </ul>
+        </section>
+        <section class="work-container">
+            <article class="work-item">
+                <header>
+                    <figure class="work-img"><img src="assets/images/project-tonic.png"
+                            alt="Project Tonic Prevew Image"></figure>
```

Comment:
- [ ] Kindly make your card images responsive by giving those images `width: 100%;` :blush: so then the images would be responsive too :+1: